src/OpenLoco/Windows/CompanyWindow.cpp

```diff
@@ -1922,6 +1935,11 @@ namespace OpenLoco::Ui::Windows::CompanyWindow
                 case Common::widx::tab_challenge:
                     Common::switchTab(self, widgetIndex);
                     break;
+
+                case widx::loan_autopay:
+                    auto company = CompanyManager::get(CompanyId(self->number));
```

Comment:
I think you'll need to add braces around this case statement due to defining a new variable